### PR TITLE
Upgrade deployer SPI and Spring Cloud Task

### LINKED ISSUE #10: Upgrade SPI and implementation versions to 1.2
As a developer, I'd like to upgrade SPI and CF-deployer implementation to the latest 1.2 release.

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- also upgrade reactor-core version

Resolves #10

### CODE REVIEW

File: tasklauncher-cloudfoundry-app-dependencies/pom.xml [hunk 1]
```diff
@@ -13,15 +13,15 @@
     <parent>
         <artifactId>spring-cloud-dependencies-parent</artifactId>
         <groupId>org.springframework.cloud</groupId>
-        <version>1.2.1.RELEASE</version>
+        <version>1.2.1.BUILD-SNAPSHOT</version>
```
Comment (cppwfs) [inline tasklauncher-cloudfoundry-app-dependencies/pom.xml:16]: Just curious why we wouldn't go ahead and uses either the 1.3.2.RELEASE or the 1.3.3.BUILD-SNAPSHOT?

Comment (ilayaperumalg) [reply tasklauncher-cloudfoundry-app-dependencies/pom.xml:16]: Good point. I think we can apply this change across all the app starters may be?

**Comment by sabbyanandan** [comment]
@ilayaperumalg @cppwfs: Ping. It'd be great to get this merged soon.

File: tasklauncher-cloudfoundry-app-dependencies/pom.xml (hunk 1 above)
Comment (ilayaperumalg) [reply tasklauncher-cloudfoundry-app-dependencies/pom.xml:16]: @cppwfs What do you think about merging without the `1.3.x` upgrade as we can update it across all the projects if we want to do so?

Comment (cppwfs) [reply tasklauncher-cloudfoundry-app-dependencies/pom.xml:16]: sounds good.

**Comment by cppwfs** [comment]
LGTM
Will update task and deployer version to 1.2.1 on merge.

**Comment by cppwfs** [comment]
Updated to 1.2.1 .  squashed merged.

**Pull request closed without merging**